Source/WebCore/platform/graphics/coretext/FontCoreText.cpp

```diff
@@ -780,6 +780,22 @@ FloatRect Font::platformBoundsForGlyph(Glyph glyph) const
     return boundingBox;
 }
 
+Vector<FloatRect> Font::platformBoundsForGlyphs(const Vector<Glyph>& glyphs) const
+{
+    Vector<CGRect> rectsForGlyphs(glyphs.size());
+    auto ignoredBoundingBox = CTFontGetBoundingRectsForGlyphs(getCTFont(), platformData().orientation() == FontOrientation::Vertical ? kCTFontOrientationVertical : kCTFontOrientationHorizontal, glyphs.data(), rectsForGlyphs.data(), rectsForGlyphs.size());
+    (void)ignoredBoundingBox;
+
+    auto boundingBoxes = rectsForGlyphs.map([&](const CGRect& cgrect) -> FloatRect {
```

Comment:
nit: I think we should be able to deduce type `CGRect` here with `auto` for the lambda argument